contexts/analytics.js

```diff
@@ -0,0 +1,117 @@
+import { MatomoProvider, createInstance, useMatomo } from '@datapunt/matomo-tracker-react'
```

Comment:
@Jonasslv or @saito-sv you have anything thoughts about this new analytics page created?